### PR TITLE
Fixes #29

### LINKED ISSUE #29: Forbidden channels
When selecting a server (but no channel), you can send messages (in my case, they show up on the default #general channel) but you don't see any messages in the list (because no channels are selected).

I suggest that if a server is selected, but not a channel, a default channel should be selected for viewing as well as for sending messages.

Thanks for the project!

### PR DESCRIPTION
**Pull request by glitchassassin**

### CODE REVIEW

File: discurses/ui/message_list.py [hunk 1]
```diff
@@ -401,6 +405,35 @@ def update_columns(*args, **kwargs):
...
+            "                ||----w |                                      \n"
+            "                ||     ||                                      \n"
+            "                                                               \n"
+            "                                                               \n"
+            "Oops! This channel is Forbidden. You don't have access. Sorry!\n\n",
+            align=urwid.CENTER)
+        w = urwid.Padding(txt, left=5, right=5)
+        self.__super.__init__(w)
+
+    def update_columns(*args, **kwargs):
```
Comment (Ge0) [inline discurses/ui/message_list.py]: What is this method for?

Comment (topisani) [reply discurses/ui/message_list.py]: See line 87. This widget acts as a MessageWidget, and those need an update_columns function.

Comment (Ge0) [reply discurses/ui/message_list.py]: I see. If there is nothing to do in this `update_columns` method, I think that you can discard it in the child class.

Comment (topisani) [reply discurses/ui/message_list.py]: There is no common `MessageLikeWidget` superclass, it uses duck typing

Comment (Ge0) [reply discurses/ui/message_list.py]: Oh I see! Let's keep this then, and consider refactoring the widget's hierarchy in the future.

Comment (glitchassassin) [reply discurses/ui/message_list.py]: Fixed the PEP-8 issues, now passes CI.

Comment (Ge0) [reply discurses/ui/message_list.py]: Thanks a lot! I'll try to test to feature asap & merge! ;)

Comment (Ge0) [reply discurses/ui/message_list.py]: Mmh, I don't know how you can connect to a forbidden channel through discurses. Is there any way to manually type a chan name that is hidden (& forbidden) to reach your code?

Comment (topisani) [reply discurses/ui/message_list.py]: if its the default channel - see #29

Comment (glitchassassin) [reply discurses/ui/message_list.py]: When I joined a server with a Forbidden channel, it showed up in the channel list in the server tree.

Comment (glitchassassin) [reply discurses/ui/message_list.py]: It wasn't the default channel on the server, in this case.

Comment (Ge0) [reply discurses/ui/message_list.py]: Ok

@topisani I can't test @glitchassassin atm. Shall I merge still?

**Pull request merged**